lib/routes/Character/components/CharacterDialog.tsx

```diff
@@ -612,6 +613,31 @@ export const CharacterDialog: React.FC<{
     );
   }
 
+  function renderNotes() {
+    return (
+      <>
+        {renderSheetHeader(
+          characterManager.state.character?.notesLabel ??
+            t("character-dialog.notes"),
+          characterManager.actions.setNotesLabel
+        )}
+        <Box className={sheetContentStyle}>
+          <Typography>
+            <ContentEditable
+              border
+              readonly={props.readonly}
+              value="foobar"
```

Comment:
must: here the value of the ContentEdiable component is `foobar` instead of `characterManage.state.notes`

Also, we should do 

```
value={characterManager.state.notes || ""} 
```

since it can be undefined (like I mentioned above)